### ISSUE TITLE
3-px error is always 0.000 on own dataset

### ISSUE THREAD

**Issue by hubery4** [JiaRenChang/PSMNet#220]
Hi @JiaRenChang ,

When I use my dataset to train the network, the 3-px error is always 0.000 and the output disparity is black using the model.
Do you have any ideas to handle this problem?

Thank you very much.

**Comment by hubery4** [comment]
> Hi @JiaRenChang ,
> 
> When I use my dataset to train the network, the 3-px error is always 0.000 and the output disparity is black using the model.
> Do you have any ideas to handle this problem?
> 
> Thank you very much.
@JiaRenChang @mileyan @H1Gdev 
Also, when I use my own data to train main.py, the loss value will immediately converge and remain unchanged. 
![image](https://user-images.githubusercontent.com/47403404/125177545-9be93380-e1d4-11eb-8085-60d2ec2d237e.png)
Then, when I test my own image use the pretrained model, the disparity output will be black.

Do you know how to fix it?

Thank you very much.

**Comment by JiaRenChang** [comment]
@hubery4 
   You can print your output tensor to see the values.
   If the values seem correctly, the black disparity may be caused by saving .png.
   You can change the save png tools, such as cv2.
 Jia-Ren

**Comment by zzr0127** [comment]
你好，我也出现这个问题了，现在这个问题集体怎么就解决

**Comment by zzr0127** [comment]
Hello, I also have this problem, how to solve it？

**Comment by zzr0127** [comment]
> 你好，这个问题你解决了吗，我也出现这个问题了
